sdk/horizon/streamer.go

```diff
@@ -60,12 +61,21 @@ func (h *Streamer) streamTx(cursor string, txs chan<- agent.StreamedTransaction,
 			Cursor: cursor,
 		}
 		err := h.HorizonClient.StreamTransactions(ctx, req, func(tx horizon.Transaction) {
+			txOrderedID, err := strconv.ParseInt(tx.PagingToken(), 10, 64)
+			if err != nil {
+				ctxCancel()
+				if h.ErrorHandler != nil {
+					h.ErrorHandler(err)
+				}
+				return
+			}
 			cursor = tx.PagingToken()
 			streamedTx := agent.StreamedTransaction{
-				Cursor:         cursor,
-				TransactionXDR: tx.EnvelopeXdr,
-				ResultXDR:      tx.ResultXdr,
-				ResultMetaXDR:  tx.ResultMetaXdr,
+				Cursor:             cursor,
+				TransactionOrderID: txOrderedID,
```

Comment:
💭 can we make them all `OrderID` instead of `OrderedID`? For consistency, and I think `Order` makes more sense than `Ordered`

Reply:
Good call, thanks for picking up on the inconsistency. 9ce4077